moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -1136,7 +1283,24 @@ bool IKFastKinematicsPlugin::getPositionIK(const geometry_msgs::Pose& ik_pose, c
   }
 
   KDL::Frame frame;
-  tf::poseMsgToKDL(ik_pose, frame);
+  if (transform_tip_ || transform_base_)
```

Comment:
add a comment explaining this code section

Reply:
done